src/player.h

```diff
@@ -1201,6 +1201,8 @@ class player : public Character, public JsonSerializer, public JsonDeserializer
         void set_underwater(bool);
         bool is_hallucination() const override;
 
+        bool is_driving() const; /** Is the player currently controlling a moving vehicle */
```

Comment:
Why is this part of the player interface? It's only used in player.  Minimally default new methods to private unless they are called from outside the class, or ideally make it a static method in ranged.cpp that takes a reference to player.